addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentCard.js

```diff
@@ -34,12 +40,15 @@ class ScEnvironmentCard extends React.Component {
   render() {
     const env = this.environment;
     const state = env.state;
+    const indexesStore = this.indexesStore;
```

Comment:
is `indexesStore` and `indexes` being used?

Reply:
Removed